apps/api/src/app/app.module.ts

```diff
@@ -42,6 +43,12 @@ const autoSchemaFile = debug ? 'apps/api/src/api.graphql' : true
     DocumentModule,
     TranslationsModule,
     TerminusModule,
+    NationalRegistryModule.register({
+      baseSoapUrl: process.env.SOFFIA_SOAP_URL,
```

Comment:
I think you should not be using the `process.env` but the `environment` abstraction

Reply:
By `environment` I mean something like [this](https://github.com/island-is/island.is/blob/2333104719e5cc727e324744937e5d76d07b5ba1/apps/gjafakort/api/src/environments/environment.prod.ts).